### PR TITLE
64815 - [SW] - Review page resolution options enhancements

### PR DESCRIPTION
**Pull request by OptionSelect**
## Summary
Update review page resolution option display logic per new design. Also removed unused method.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#64815

## Screenshots
Screenshot 2024-01-19 at 1 19 56 PM

## What areas of the site does it impact?
SW / FSR

### Quality Assurance & Testing

- [ X ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ X ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ X ] Linting warnings have been addressed
- [ X ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ X ] Screenshot of the developed feature is added
- [ X ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ X ] Browser console contains no warnings or errors.
- [ X ] Events are being sent to the appropriate logging solution
- [ X ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ X ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/financial-status-report/components/shared/CustomResolutionOptionReview.jsx [hunk 1]
```diff
@@ -1,25 +1,98 @@
...
+
+      <br />
     </div>
   );
 };
 
 CustomResolutionOptionReview.propTypes = {
   children: PropTypes.object,
+  data: PropTypes.object,
+  pagePerItemIndex: PropTypes.number,
```
Comment (Dr-Pongo) [inline src/applications/financial-status-report/components/shared/CustomResolutionOptionReview.jsx:95]: Might want to set this to string or number bc it gets mad in the console and makes some noise

File: src/applications/financial-status-report/pages/resolution/resolutionOption.js [hunk 2]
```diff
@@ -10,7 +9,6 @@ export const uiSchema = {
       'ui:validations': [validateResolutionOption],
       resolutionOption: {
```
Comment (Dr-Pongo) [inline src/applications/financial-status-report/pages/resolution/resolutionOption.js:10]: I'm a nervous bee, but do we want to use `ui:options` and explicitly set `hideOnReview: true,` ?
Deffo not merge blocking, just a thought!

File: src/applications/financial-status-report/utils/helpers.js
Comment (Dr-Pongo) [inline src/applications/financial-status-report/utils/helpers.js:1]: Nice catch! Esp if it wasn't being used anywhere! Thanks for keeping an eye out for housekeeping!

**Review by Dr-Pongo — DISMISSED** [review]
LGTM! 🚀 
No blocking change requests, just some neurotic notes!

**Comment by Dr-Pongo** [comment]
Last thing I just noticed is the `Personal Statement` section still reads `Resolution comment`

**Comment by OptionSelect** [comment]
image

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀 
Great work!

**Pull request merged**